git-tag/action.yml

```diff
@@ -0,0 +1,149 @@
+# Copyright 2025 Chainguard, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+name: 'GitHub Action to Bump and Push Git Tag (SemVer)'
+description: "A GitHub Action to automatically bump and push a Git tag according to the SemVer 2.0.0 specification."
+
+inputs:
+  bump_level:
+    description: "Specifies which part of the SemVer version to increment: major, minor, bugfix, prerelease, or build."
+    required: true
+    default: 'build'
+  git_tag_prefix:
+    description: "Git tag prefix (v by default)"
+    required: true
+    default: "v"
+  forced_version:
+    description: "Forced version tag to create instead of bumping from the latest version. 
+                  Must be SemVer2 compatible and not already. Do not specify the prefix."
+    required: false
+  dry_run:
+    required: true
+    type: boolean
+    description: "If true, no changes will be pushed 
+                  (useful for testing the workflow without creating a tag)."
+    default: false
+  github_token:
+    description: "GitHub token required to push the tag"
+    required: false
+  use-gitsign:
+    description: "Use gitsign to sign commits."
+    default: 'true'
+    required: false
+
+outputs:
+    bumped_version:
+      description: "Version bumped"
+      value: ${{ steps.semver-bump.outputs.bumped_version }}
+    git_tag:
+      description: "Git tag"
+      value: ${{ steps.semver-bump.outputs.git_tag }}
+
+runs:
+  using: "composite"
+  steps:
+    # Semantic Versioning parsing and manipulation tool
+    - name: Download semver-tool and verify checksum
+      id: download-semver-tool
+      shell: bash
+      run: |
+        set -e
+
+        SEMVER_URL="https://raw.githubusercontent.com/fsaintjacques/semver-tool/3.4.0/src/semver"
```

Comment:
we should bump to the hash and not to the tag

use the hash of the tag and add a comment saying we extracted from the tag 3.4.0